HyperVExtension/src/HyperVExtension/Strings/en-US/Resources.resw

```diff
@@ -154,7 +154,7 @@
     <comment>Error text for when we fail to create the file that the disk image will be downloaded into</comment>
   </data>
   <data name="ErrorPerformingOperation" xml:space="preserve">
-    <value>Unable to perform the requested operation. Check the Dev Home Hyper-V extension's log files for more information.</value>
+    <value>Unable to perform the requested operation. Check the Dev Home Hyper-V extension's log file: {0} for more information.</value>

```

Comment:
"Dev Home Hyper-V Extension" is the name (right?) so I think the E should be capitalized.